app/views/deals/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1 class="new-form">New transaction</h1>
+
+<%= render "form", deal: @deal %>
+
+<br>
+
+<div class="back">
+  <%= link_to "Back to transactions", category_path(id: @category.id) %>
+</div>
```

Comment:
- [ ] Please place the "Back" button (<) at the top navigation of the page just as in the Behance design shown below 👇 
### Your Design
![image](https://github.com/muhammedb8826/budget-app/assets/46839548/3935a112-e84b-42ba-aec9-898731a03a99)
### Design Guideline
![image](https://github.com/muhammedb8826/budget-app/assets/46839548/b8dfec98-2e56-4003-9b2c-0f9903dce92e)